app/Config/Database.php

```diff
@@ -62,7 +63,7 @@ class Database extends Config
         'username' => '',
         'password' => '',
         'database' => ':memory:',
-        'DBDriver' => 'SQLite3',
+        'DBDriver' => SQLite3::class,
```

Comment:
I am uncomfortable with this. It is just a string, not a classname.

Reply:
I reverted `SQLite3` as `"SQLite3"` string https://github.com/codeigniter4/CodeIgniter4/pull/5813/commits/6a0ed54b9922c7c88dce16033615750d7fe67d36